recipes/ocp/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "ocp" %}
+{% set version = "7.5.3" %}
+{% set occt_ver = "7.5.3" %}
```

Comment:
Is there a reason for having two variables here?

Reply:
The package version and occt version are different, so I will leave this as-is